packages/server/src/plugin/landingPage/default/getEmbeddedHTML.ts

```diff
@@ -121,8 +129,24 @@ id="embeddableSandbox"
       variables: config.variables ?? undefined,
       headers: config.headers ?? undefined,
       includeCookies: config.includeCookies ?? undefined,
+      collectionId: config.defaultOperation?.collectionId ?? undefined,
+      operationId: config.defaultOperation?.operationId ?? undefined,
```

Comment:
`?? undefined` will be redundant with the typings change I've suggested below

```suggestion
      collectionId: config.defaultOperation?.collectionId,
      operationId: config.defaultOperation?.operationId,
```